index.js

```diff
@@ -1,5 +1,8 @@
 //
 // Setup and Execution
+
+const { DateTime } = require('luxon');
```

Comment:
Also the formatting here is broken. `const { DateTime } = require('luxon');` should come after the trailing `//`. Also don't forget to leave a newline between it and `const config`.

Reply:
This was auto inserted, for styling we need to add eslint if we want to be this strict because manually reviewing formatting is not manageable.

Reply:
That's more work. For now, we need to fix it manually and we'll add it later.